File: src/main/java/com/flowcommerce/core/storage/member/SellerRepository.java
package com.flowcommerce.core.storage.member;

import org.springframework.data.jpa.repository.JpaRepository;

public interface SellerRepository extends JpaRepository<SellerEntity, Long> {
    boolean existsBySellerName(String sellerName);
    boolean existsByBusinessNumber(String businessNumber);
    SellerEntity findByMemberId(Long memberId);
}

File: src/main/kotlin/com/flowcommerce/core/storage/member/SellerRepository.kt
package com.flowcommerce.core.storage.member

import org.springframework.data.jpa.repository.JpaRepository

interface SellerRepository : JpaRepository<SellerEntity, Long> {
    fun existsBySellerName(sellerName: String): Boolean
    fun existsByBusinessNumber(businessNumber: String): Boolean
    fun findByMemberId(memberId: Long): SellerEntity?
}
